test/test.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const assert = require('assert');
+const spawn = require('child_process').spawn;
+const exec = require('child_process').exec;
+const request = require('request');
+const util = require('util');
+
+describe('nodejs-docker', () => {
+
+  it('serves traffic on 8080', (done) => {
+    runDocker('test/express', 8080, (host, callback) => {
+      setTimeout(() => {
+        request(`http://${host}:8080`, (err, res, body) => {
+          if (err) {
+            console.error(`Error requesting content: ${util.inspect(err)}`);
+            throw err;
+          }
+          assert.equal(body, 'Hello World!');
+          callback(() => {
+            done();
+          });
+        });
+      }, 3000);
```

Comment:
Is there a better way to do this?  Waiting for the container to fire up before making the request, otherwise it fails.